public/scripts/components/Navigation.js

```diff
@@ -1,11 +1,47 @@
 import React from 'react';
+import {Link} from 'react-router';
+import $ from 'jquery';
+import user from '../models/user';
+import {browserHistory} from 'react-router';
+
 
 export default React.createClass({
+	getInitialState: function() {
+		return {
+			user: user
+		};
+	},
+
+	componentDidMount: function() {
+		this.state.user.on('change', () => {
+			this.setState({
+				user: user
+			});
+		});
+	},
+
 	render: function() {
-		return (
-			<nav>
-				This is the navigation
-			</nav>
-		);
+		if(this.state.user.get('id')) {
+			return (<nav>
+			<img src="../../images/pointer_logo.png" />
```

Comment:
This line might cause problems depending on the page. I would recommend using an absolute URL (starting with a /) so that it will work on any page (eg `/images/pointer_logo.png`)